ivy/functional/backends/paddle/elementwise.py

```diff
@@ -784,21 +604,26 @@ def positive(
     return x.clone()
 
 
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
+            "int32",
+            "int64",
+            "float",
```

Comment:
have you tested if `float16` and `bfloat16` work without type casting for this function?